src/test/java/core/basesyntax/dao/FruitDaoTest.java

```diff
@@ -0,0 +1,91 @@
+package core.basesyntax.dao;
+

```

Comment:
I think it's FruitDaoImplTest, cause we need to test realization, not the abstraction

Reply:
and also isn't it excessive? cause we had to test our services, not dao =)